cmd/neofs-node/config.go

```diff
@@ -131,6 +132,13 @@ type cfg struct {
 	persistate *state.PersistentStorage
 
 	netMapSource netmapCore.Source
+
+	// current network map
+	netMap atomicstd.Value // type netmap.NetMap
+}
+
+func (c *cfg) ProcessCurrentNetMap(f func(netmap.NetMap)) {
```

Comment:
why not just getter?

Reply:
The need is to perform exact single action. We could implement it as a getter, doesn' matter.

Reply:
I also think `CurrentNetmap() netmap.NetMap` is simpler to read and use.

Reply:
me too

Reply:
Finally I've decided to leave `netmap` service package a dummy message constructor. All the business logic is implemented on application side. 0e7bc5d0cf2f29b24849d6ebb45e110f914614ed